packages/bs-neo-legacy/src/services/BSNeoLegacy.ts

```diff
@@ -120,6 +121,77 @@ export class BSNeoLegacy<BSName extends string = string>
     this.claimToken = tokens.find(token => token.symbol === 'GAS')!
   }
 
+  #transactionHasMoreThanMaxSize(config: any) {
+    if (!config.fees || config.fees < BSNeoLegacyConstants.FEE_APPLIED_TO_PLAYABLE_TRANSACTION) {
+      const serializedTransaction = config.tx.serialize(true)
+      const transactionSize = serializedTransaction.length / 2
+
+      if (transactionSize > BSNeoLegacyConstants.MAX_TRANSACTION_SIZE_WITHOUT_FEE) {
+        return true
+      }
+    }
+
+    return false
+  }
+
+  async #signTransfer(config: any, nep5ScriptBuilder?: any) {
+    if (!nep5ScriptBuilder || nep5ScriptBuilder.isEmpty()) {
+      config = await api.createContractTx(config)
+    } else {
+      config.script = nep5ScriptBuilder.str
+      config = await api.createInvocationTx(config)
+    }
+    config = await api.modifyTransactionForEmptyTransaction(config)
+    config = await api.addAttributeIfExecutingAsSmartContract(config)
+    config = await api.signTx(config)
+    config = await api.addSignatureIfExecutingAsSmartContract(config)
+
+    return config
+  }
+
+  async #sendTransfer(config: any, nep5ScriptBuilder?: any) {
+    const sharedConfig = await api.fillBalance(config)
+
+    let signedConfig = await this.#signTransfer({ ...sharedConfig }, nep5ScriptBuilder)
```

Comment:
Why destructing `sharedConfig`?

Reply:
Because the `api` methods change the object that you sent to them, so I needed to make a clone